### PR TITLE
doc for dropdown, menu and toast

### PR DESCRIPTION
**Pull request by jimmymadrigal**

### CODE REVIEW

File: packages/toast/src/docs/Toast.doc.mdx [hunk 1]
```diff
@@ -1 +1,133 @@
...
+
+you can check that behavior [here](/story/ui-bootstrap-toast--custom-container)
+
+## Message configuration
+
+| Name | Type  | Description |
+| ---- | ---- | ----------- |
+| `animate` | `boolean` | if the toast message is animated |
+| `headerText` | `string` | the header of the toast message, if not present then no header is displayed |
+| `message` | `string` | actial toast message |
```
Comment (alejandronavarroGL) [inline packages/toast/src/docs/Toast.doc.mdx:68]: typo: actial -> actual

File: packages/toast/src/docs/Toast.doc.mdx [hunk 2]
```diff
@@ -1 +1,133 @@
...
+    }
+  }
+}
+```
+
+Check the page [what is a theme](/docs/docs-theming--page#what-is-a-theme) to see how to use this json
+
+## Inner components
+
+Note that the Modal uses [UiTransition](?path=/docs/ui-bootstrap-transition-default&args=#transitions) and [UiButtonclose](?path=/docs/ui-bootstrap-closebutton--default&args=#close-button) components internally, so when providing a theme you should provide for them as well.
```
Comment (alejandronavarroGL) [inline packages/toast/src/docs/Toast.doc.mdx:133]: idem: I think this is part of modal doc.

**Review by jeanpaniagua — APPROVED** [review]

**Review by alejandronavarroGL — CHANGES_REQUESTED** [review]
some comments

File: packages/dropdown/src/docs/Dropdown.doc.mdx [hunk 3]
```diff
@@ -1 +1,129 @@
...
+
+<Canvas>
+  <Story id="ui-bootstrap-dropdown--left" />
+  <Story id="ui-bootstrap-dropdown--right" />
+  <Story id="ui-bootstrap-dropdown--up" />
+</Canvas>
+
+# Popper Js and menu on the edges
+By using Popper Js the menu can accomodate itself when it gets to the edges.
+You can play around with that functionallity [here](/story/ui-bootstrap-dropdown--edge)
```
[the remark this answers is not in the document]
Comment (jimmymadrigal) [reply packages/dropdown/src/docs/Dropdown.doc.mdx]: Done

File: packages/toast/src/docs/Toast.doc.mdx (hunk 2 above)
Comment (jimmymadrigal) [reply packages/toast/src/docs/Toast.doc.mdx]: Done

File: packages/toast/src/docs/Toast.doc.mdx (hunk 1 above)
Comment (jimmymadrigal) [reply packages/toast/src/docs/Toast.doc.mdx]: Done

File: packages/dropdown/src/docs/Dropdown.doc.mdx [hunk 4]
```diff
@@ -1 +1,129 @@
...
+    }
+  }
+}
+```
+
+Check the page [what is a theme](/docs/docs-theming--page#what-is-a-theme) to see how to use this json
+
+## Inner components
+
+Note that the Modal uses [UiMenu](?path=/docs/ui-bootstrap-menu-normal&args=#menu) and [UiButton](?path=/docs/ui-bootstrap-button--primary&args=#buttons) components internally, so when providing a theme you should provide for them as well.
```
[the remark this answers is not in the document]
Comment (jimmymadrigal) [reply packages/dropdown/src/docs/Dropdown.doc.mdx]: Done

File: packages/dropdown/src/docs/Dropdown.doc.mdx [hunk 5]
```diff
@@ -1 +1,129 @@
...
+
+## Props
+
+| Name | Type  | Description |
+| ---- | ---- | ----------- |
+| `label` | `string` | button label, it overrides the slot content if present |
+| `datasource` | `array` | configuration for the (menu component](?path=/docs/ui-bootstrap-menu-normal&args=#menu) |
+| `invert` | `boolean` | inverts the menu component |
+| `outline` | `boolean` | outline class is added to the button |
+| `variant` | `string` | varian class defined in the theme |
```
[the remark this answers is not in the document]
Comment (jimmymadrigal) [reply packages/dropdown/src/docs/Dropdown.doc.mdx]: Done

**Review by alejandronavarroGL — APPROVED** [review]

**Pull request merged**